addons/mail/static/src/core/common/thread_model.js

```diff
@@ -125,13 +125,13 @@ export class Thread extends Record {
     custom_channel_name;
     /** @type {string} */
     description;
-    followers = Record.many("Follower");
+    followers = Record.many("Follower", { inverse: "followedThread" });
```

Comment:
The reason why this `inverse: "followedThread"` was not added is due to inverse of `followedThread` being both `followers` and `selfFollower`.

`followers` has to be a continuous sequence of followers, so that we know the last follower id can be used to load more. We definitely don't want to introduce holes, e.g. by loading followers `[1...100]` and inserting `selfFollower` `200`, which is likely to prevent loading followers `[101-199]`.

So I think that without changing the logic of loading more on followers, introducing this `inverse` has the risk to introduce hole above, which functionally results to preventing to load some followers in some cases. Better not introduce it for the moment

Reply:
Your comment has been taken into account. Extra follower in test is due to adding condition in get followers to not return self follower (as it is returned separately). So the amount is the limit +1.